src/Setup/BuildPlan.purs

```diff
@@ -52,19 +49,24 @@ getVersionField key = do
         pure (pure (Exact version))
 
 -- | Resolve the exact version to provide for a tool in the environment, based
--- | on the action.yml file.
-resolve :: Json -> Tool -> ExceptT Error Effect (Maybe { tool :: Tool, version :: Version })
+-- | on the action.yml file. In the case of NPM packages, bypass the action.yml
+-- | file and use 'Latest'.
+resolve :: Json -> Tool -> ExceptT Error Effect (Maybe { tool :: Tool, versionField :: VersionField })
```

Comment:
I wonder if this functionality should just be baked into the `installMethod` function so that there are fewer cases to admit? After all, that's the place where we know explicitly whether a tool is NPM-installed or not.